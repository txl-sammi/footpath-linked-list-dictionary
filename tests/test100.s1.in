Queensberry Street between Elizabeth Street and O'Connell Street
Park Drive between Bayles Street and Gatehouse Street
Franklin Street between Queen Street and William Street
Wimble Street between Morrah Street and Park Drive
Reeves Street between Rathdowne Street and Drummond Street
Queensberry Street between Cobden Street and Peel Street
Franklin Street between Victoria Street and Swanston Street
Queensberry Street between Capel Street and Howard Street
Punch Lane from Little Bourke Street
Hayward Lane between Little Lonsdale Street and La Trobe Street
Swanston Street between Elgin Street and College Crescent
Carlton Street between Nicholson Street and Canning Street
Gatehouse Street between Morrah Street and Bayles Street
Cemetery Road West between Royal Parade and College Crescent
Lonsdale Street between Exhibition Street and Russell Street
Royal Parade between Gatehouse Street and MacArthur Road
Leicester Street between Queensberry Street and Pelham Street
Lygon Street between Victoria Street and Queensberry Street
Elizabeth Street between Flemington Road and Grattan Street
Blackwood Street between Flemington Road and Courtney Street
Rathdowne Street between Pelham Street and Grattan Street
Evelyn Place between Nicholson Street and Albert Street
Elizabeth Street between Franklin Street and Therry Street
David Street between Elgin Street and Palmerston Street
Swanston Street between Lonsdale Street and Little Lonsdale Street
Franklin Street between Elizabeth Street and Queen Street
Faraday Street between Canning Street and Rathdowne Street
O'Connell Street between Queensberry Street and Peel Street
Palmerston Street between Nicholson Street and Canning Street
Cemetery Road East between College Crescent and Lygon Street
Pelham Street between Bouverie Street and Leicester Street
Morrah Street between Fitzgibbon Street and Wimble Street
Berkeley Street between Pelham Street and Grattan Street
Lygon Street between Queensberry Street and Argyle Place South
Exhibition Street between La Trobe Street and Victoria Street
Princes Park Drive between Cemetery Road West and MacPherson Street
Little La Trobe Street between Swanston Street and Elizabeth Street
Palmerston Street between Rathdowne Street and Drummond Street
Canning Street between Palmerston Street and Pitt Street
Drummond Street between Faraday Street and Elgin Street
Swanston Street between Faraday Street and Elgin Street
Intersection of Queen Street and Franklin Street
Peel Street between Queensberry Street and Elizabeth Street
Albert Street between Nicholson Street and Gisborne Street
MacArthur Place North between Canning Street and Rathdowne Street
Astor Place from Lygon Street
Palmerston Street between Lygon Street and Keppel Street
Elizabeth Street between Therry Street and Victoria Street
Elgin Street between Nicholson Street and Canning Street
Queensberry Street between Drummond Street and Lygon Street
Victoria Parade between Nicholson Street and Gisborne Street
Nicholson Street between Palmerston Street and Kay Street
Royal Parade between Grattan Street and Story Street
Cathedral Place between Lansdowne Street and Gisborne Street
Queensberry Street between Lygon Street and Cardigan Street
Drummond Street between Pelham Street and Grattan Street
O'Connell Street between Victoria Street and Queensberry Street
University Street between Rathdowne Street and Drummond Street
A'Beckett Street between Wills Street and Queen Street
Little Barkly Street between Faraday Street and Rathdowne Street
Lygon Street between Argyle Place South and Pelham Street
Victoria Street between Bouverie Street and Elizabeth Street
