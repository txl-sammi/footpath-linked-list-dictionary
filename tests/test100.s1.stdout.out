Queensberry Street between Elizabeth Street and O'Connell Street --> 1
Park Drive between Bayles Street and Gatehouse Street --> 1
Franklin Street between Queen Street and William Street --> 1
Wimble Street between Morrah Street and Park Drive --> 1
Reeves Street between Rathdowne Street and Drummond Street --> 1
Queensberry Street between Cobden Street and Peel Street --> 2
Franklin Street between Victoria Street and Swanston Street --> 1
Queensberry Street between Capel Street and Howard Street --> 1
Punch Lane from Little Bourke Street --> 1
Hayward Lane between Little Lonsdale Street and La Trobe Street --> 1
Swanston Street between Elgin Street and College Crescent --> 1
Carlton Street between Nicholson Street and Canning Street --> 1
Gatehouse Street between Morrah Street and Bayles Street --> 1
Cemetery Road West between Royal Parade and College Crescent --> 1
Lonsdale Street between Exhibition Street and Russell Street --> 2
Royal Parade between Gatehouse Street and MacArthur Road --> 1
Leicester Street between Queensberry Street and Pelham Street --> 1
Lygon Street between Victoria Street and Queensberry Street --> 2
Elizabeth Street between Flemington Road and Grattan Street --> 1
Blackwood Street between Flemington Road and Courtney Street --> 1
Rathdowne Street between Pelham Street and Grattan Street --> 1
Evelyn Place between Nicholson Street and Albert Street --> 1
Elizabeth Street between Franklin Street and Therry Street --> 1
David Street between Elgin Street and Palmerston Street --> 2
Swanston Street between Lonsdale Street and Little Lonsdale Street --> 1
Franklin Street between Elizabeth Street and Queen Street --> 1
Faraday Street between Canning Street and Rathdowne Street --> 1
O'Connell Street between Queensberry Street and Peel Street --> 1
Palmerston Street between Nicholson Street and Canning Street --> 1
Cemetery Road East between College Crescent and Lygon Street --> 1
Pelham Street between Bouverie Street and Leicester Street --> 1
Morrah Street between Fitzgibbon Street and Wimble Street --> 1
Berkeley Street between Pelham Street and Grattan Street --> 1
Lygon Street between Queensberry Street and Argyle Place South --> 1
Exhibition Street between La Trobe Street and Victoria Street --> 1
Princes Park Drive between Cemetery Road West and MacPherson Street --> 1
Little La Trobe Street between Swanston Street and Elizabeth Street --> 1
Palmerston Street between Rathdowne Street and Drummond Street --> 1
Canning Street between Palmerston Street and Pitt Street --> 1
Drummond Street between Faraday Street and Elgin Street --> 1
Swanston Street between Faraday Street and Elgin Street --> 1
Intersection of Queen Street and Franklin Street --> 1
Peel Street between Queensberry Street and Elizabeth Street --> 1
Albert Street between Nicholson Street and Gisborne Street --> 1
MacArthur Place North between Canning Street and Rathdowne Street --> 1
Astor Place from Lygon Street --> 1
Palmerston Street between Lygon Street and Keppel Street --> 1
Elizabeth Street between Therry Street and Victoria Street --> 1
Elgin Street between Nicholson Street and Canning Street --> 1
Queensberry Street between Drummond Street and Lygon Street --> 1
Victoria Parade between Nicholson Street and Gisborne Street --> 1
Nicholson Street between Palmerston Street and Kay Street --> 1
Royal Parade between Grattan Street and Story Street --> 1
Cathedral Place between Lansdowne Street and Gisborne Street --> 1
Queensberry Street between Lygon Street and Cardigan Street --> 1
Drummond Street between Pelham Street and Grattan Street --> 1
O'Connell Street between Victoria Street and Queensberry Street --> 1
University Street between Rathdowne Street and Drummond Street --> 1
A'Beckett Street between Wills Street and Queen Street --> 1
Little Barkly Street between Faraday Street and Rathdowne Street --> 1
Lygon Street between Argyle Place South and Pelham Street --> 1
Victoria Street between Bouverie Street and Elizabeth Street --> 1
